wp-content/themes/reconasia/assets/_scss/components/_post-block.scss

```diff
@@ -1,22 +1,92 @@
 @use 'abstracts' as *;
 
 .post-block {
+  display: grid;
+  grid-template-areas:
+    'img'
+    'title'
+    'excerpt'
+    'authors'
+    'date'
+    'categories';
+  grid-template-columns: 1fr;
+  margin-top: rem(40);
+  margin-right: auto;
+  margin-left: auto;
+
+  &.has-post-thumbnail {
+    @include breakpoint('medium') {
+      grid-template-areas:
+        'title img'
+        'excerpt img'
+        'authors img'
+        'date img'
+        'categories img';
+      grid-template-columns: 2fr 1fr;
+      grid-template-rows: max-content max-content max-content max-content auto;
+      column-gap: rem(32);
+    }
+  }
+
+  &__img {
+    margin-bottom: rem(24);
+  }
+
   &__title {
-    color: $color-black-100;
+    grid-area: title;
+    color: $color-black-190;
+    @extend %text-style-heading-large-serif-bold;
 
     a {
-      &:visited {
-        // color: $color-text-gray-70;
-      }
-
       &:hover {
-        // color: $color-primary-base;
+        color: $color-link-primary-200;
+        transition: color 0.3s ease-in-out;
```

Comment:
```suggestion
```
You don't need to include this, as the styles set on all `a` tags in `_typography.scss` set the transition. :)